ui-admin/src/portal/MailingListView.tsx

```diff
@@ -118,11 +114,14 @@ export default function MailingListView({ portalContext, portalEnv }:
           <RowVisibilityCount table={table}/>
         </div>
         <div className="d-flex">
-          <Button onClick={download}
-            variant="light" className="border m-1" disabled={!numSelected}
-            tooltip={numSelected ? 'Download selected contacts' : 'You must select contacts to download'}>
-            <FontAwesomeIcon icon={faDownload} className="fa-lg"/> Download
```

Comment:
I switched download to use the generic DownloadControl component from tableUtils. It does introduce a behavior change. Does an admin actually need to be able to select individual contacts to download or is downloading the entire table fine?

Reply:
The reason we supported select+download was in case they wanted to filter the list (only email people who signed up this month?).  But I'm not sure we have any actual use case for that, so getting rid of it is fine.